koans/VectorOperationKoans.jl

```diff
@@ -1,2 +1,37 @@
 module VectorOperationKoans
+using LinearAlgebra
+
+function vector_lenght(x)
+end
+
+function range_1_to_n(n)
```

Comment:
These exercises would really benefit from having a small description of the problem to solve, along with a few examples for each.